lib/v2/abskoop/index.js

```diff
@@ -7,13 +7,12 @@ const { parseDate } = require('@/utils/parse-date');
 module.exports = async (ctx) => {
     const response = await got({
         method: 'get',
-        url: 'https://www.abskoop.com/page/1/',
+        url: 'https://www.ahhhhfs.com/page/1/',
```

Comment:
This url will be redirect to `https://www.ahhhhfs.com`, please use it directly.

Reply:
fixed

Reply:
I'm sorry, I misunderstood. It's been fixed